.github/CODEOWNERS

```diff
@@ -90,4 +90,11 @@
 /tasks/system-probe.py                  @DataDog/burrito
 /tasks/trace.py                         @DataDog/apm-agent
 
+/test/benchmarks                        @DataDog/agent-core
+/test/e2e                               @DataDog/container-integrations
+/test/integration                       @DataDog/container-integrations
```

Comment:
Shouldn't this be integrations?

Reply:
yeah, it should

Reply:
the naming is a bit confusing, but the tests in `/test/integration` have nothing to do with Agent integrations, they're _integration_ tests (as opposed to _unit_ tests) of the Agent. I think `agent-all` makes sense here.

Reply:
(or `agent-core`, or `agent-platform`, if we want precise responsibilities to be defined)